source/_posts/2022-02-08-android-february.markdown

```diff
@@ -0,0 +1,116 @@
+
+Why the persistent notification? It is to let Android know the app has important ongoing tasks where we do not want the background process stopped. In order for the app and its services to continue working in the background, we need to create the notification. The purpose of this notification is identical to our Sensor Worker, that allows the app to continue sending sensor updates while in the background. Just like Sensor Worker, our WebSockets notification also has its own notification channel to allow you to customize the appearance. You are free to minimize (or silence) this notification to hide the icon from appearing in your status bar. Some users may also decide to turn off the channel. While there is nothing preventing you from turning it off, if you run into connection issues you will be asked to turn the channel back on.
+
+![Screenshot of websocket settings](/images/blog/2022-02-android/websocket_setting.png)
+
+The persistent notification will contain an actionable button taking you directly to the settings page. From here you can adjust the WebSocket connection behavior and modify the notification channel. The persistent notification will only be present when an active connection to your server is being maintained.
+
+Users who are on the [minimal version] of the app will have the default WebSocket connection set to "Always" as this is the only way the app can receive notifications from your server. All other users will have the default setting "While Screen On", where the app will continue to receive notifications from Firebase when the WebSocket connection is not active. If you are on the minimal version you will need to be on Home Assistant Core 2022.2 or later in order to use this feature.
+
+We have seen varying reports from users who either do not notice much of a change in terms of battery usage while others have noticed a big difference. Please feel free to adjust the settings based on your usage and desire for the feature. Some of us developers have opted to keep the option set to "Always" and do not notice much of a change in battery usage. Personally I keep mine on "Always" and I have not had to change my daily charging habits to enjoy this feature, your experience may be different.
+
+A big thank you to [JBassett] for kicking off this feature and working hard to solve all the reported beta issues! Thank you to [jpelgrom], [SkechyWolf] and [dshokouhi] for helping out in finalizing the user experience.
+
+### Sensor Changes
+
+The remaining breaking changes in this release are related to 2 sensors offered by all versions of the app.
+
+*  [Do Not Disturb Sensor] - This sensor, which was previously offered to all supported Android versions, has been updated to use the official Android API as opposed to the undocumented method being done. The Android API is only available on devices running Android 6+ so if you are not on any supported versions you will no longer see this sensor in the Manage Sensors screen.
```

Comment:
```suggestion
*  [Do Not Disturb Sensor] - No longer available on devices running Android 5 or lower. This sensor, which was previously offered to all supported Android versions, has been updated to use the official Android API instead of the undocumented one. The Android API is only available on devices running Android 6+. Older devices will no longer see this sensor in the Manage Sensors screen.
```